calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml

```diff
@@ -55,6 +55,10 @@ spec:
       - 166.9.30.45/32 
       - 166.9.28.159/32 
       - 166.9.32.117/32
+      # New IP addresses added on 10/19/2023
```

Comment:
Since these new ones are specific to eu-es, it probably makes more sense to put them below in the `# IPs for eu-es` section (instead of this eu-de section)